CLAUDE.md

```
@@ -1,914 +1,195 @@
-4. Address any issues reported
-
-**Note**: CodeRabbit CLI must be installed via the official installer or Homebrew. It provides review suggestions only, not automatic fixes.
-
-**Installation**:
-- Official installer: `curl -fsSL https://cli.coderabbit.ai/install.sh | sh`
-- macOS via Homebrew: `brew install --cask coderabbit`
+**BEFORE EVERY ACTION: USE MCP SERVER SERENA! Activate project if not available.**
 
+When providing commit messages, never include attribution.
 
 ## Project Structure
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)